packages/map-template/README.md

```diff
@@ -62,4 +62,6 @@ To have any Stencil component changes be reflected in this project, you need to
 |`venue`|`string`|The Venue to load from your MapsIndoors Solution. |
 |`locationId`|`string`|Set a MapsIndoors Location ID to show it on the map and its details in the sheet. |
 |`primaryColor`|`string`|The primary color to use throughout the app. |
-|`logo`|`string`|The logo to show during initial load. |
\ No newline at end of file
+|`logo`|`string`|The logo to show during initial load. |
+|`appUserRoles`|`array`|Array of app user roles which can restrict/enable different features. |
```

Comment:
Isn't it an array of app user role names that need to be provided? Not just app user role objects.

```suggestion
|`appUserRoles`|`array`|Array of app user role names which can restrict/enable different Locations on the map. |
```